components/program_manager/lib/program_manager/engine.rb

```diff
@@ -3,13 +3,13 @@ class Engine < ::Rails::Engine
     config.generators do |g|
       g.test_framework :rspec
       g.fixture_replacement :factory_bot
-      g.factory_bot dir: "spec/factories"
+      g.factory_bot dir: ProgramManager::Engine.root.join('specs', 'factories').to_path
```

Comment:
I noticed that we added this folder in the plural while the standard is in the singular `spec`. I'd consider renaming it to the singular to keep it consistent with rspec.